v2/src/core/components/TheAppFooter.vue

```diff
@@ -0,0 +1,182 @@
+                                <div class="mt-5">
+                                    <div class="mb-2">
+                                        <a href="#" class="d-inline-flex align-center primary--text">
+                                            <v-img
+                                                :src="require('@/assets/icon-ethereum.svg')"
+                                                alt="Ethereum symbol in circle with purple background"
+                                                width="32"
+                                                height="32"
+                                                class="mr-3"
+                                            />
+                                            <p>Ethereum Donation</p>
+                                        </a>
+                                    </div>
+                                    <div>
+                                        <a href="#" class="d-inline-flex align-center primary--text">
+                                            <v-img
+                                                :src="require('@/assets/icon-bitcoin.svg')"
+                                                alt="Bitcoin symbol in circle with yellow background"
+                                                width="32"
+                                                height="32"
+                                                class="mr-3"
+                                            />
+                                            <p>Bitcoin Donation</p>
+                                        </a>
+                                    </div>
+                                </div>
+                            </v-col>
+                        </v-row>
+                    </v-col>
+                    <v-spacer></v-spacer>
+                    <v-col cols="12" sm="4" md="3" lg="2" class="text-lg-right mb-11 mb-lg-0">
+                        <a href="#" class="footer-item-title d-block text-h5 font-weight-bold mb-2">
+                            Settings
+                            <v-icon class="ml-2">settings</v-icon>
+                        </a>
+                        <p class="mb-3">Pricing from Coingecko</p>
+                        <div class="d-flex font-weight-light align-center justify-lg-end">
+                            <p class="text-uppercase mb-0 mr-2 text-caption font-weight-light">Powered by</p>
+                            <div class="d-flex align-center">
+                                <v-img :src="require('@/assets/logo/mew.svg')" width="24" height="24" contain class="mr-1" />
+                                <p class="text-uppercase text-h3">MEW</p>
+                            </div>
+                        </div>
+                    </v-col>
+                </v-row>
+                <div class="d-flex justify-lg-end flex-wrap pa-2">
+                    <a v-for="(link, index) in socialIcons" :key="index" :href="link.link" class="text-link">
```

Comment:
can you set target to blank so new tab gets opened up instead fo opening in the same window